chaco/tests/test_plot_factory.py

```diff
@@ -0,0 +1,68 @@
+
+from unittest import TestCase
+import numpy as np
+
+from chaco.plot_factory import create_bar_plot, create_line_plot, \
+    create_polar_plot, create_scatter_plot
+from chaco.api import BarPlot, LinePlot, PlotAxis, PlotGrid, ScatterPlot
+from chaco.polar_line_renderer import PolarLineRenderer
+
+x = np.array([1, 2, 3, 4])
+
+y = np.array([1, 2, 3, 4])
+
+
+class BaseTestRenderer(object):
```

Comment:
You can drop the `object` here -- we're no longer explicitly supporting Python 2.